hacking/test-module

```diff
@@ -64,7 +64,8 @@ def parse():
         help="path to python debugger (e.g. /usr/bin/pdb)")
     parser.add_option('-I', '--interpreter', dest='interpreter',
         help="path to interpreter to use for this module (e.g. ansible_python_interpreter=/usr/bin/python)",
-        metavar='INTERPRETER_TYPE=INTERPRETER_PATH')
+        metavar='INTERPRETER_TYPE=INTERPRETER_PATH',
+        default="ansible_python_interpreter=/usr/bin/python")
```

Comment:
maybe `sys.executable` makes most sense here as the default, so it'd use the Python that's running `test-module`?

Reply:
thanks, that indeed makes most sense, updated